src/openboardview/PDFBridge/PDFBridgeSumatra.cpp

```diff
@@ -139,6 +139,38 @@ bool PDFBridgeSumatra::InitializeDDE() {
 	return true;
 }
 
+bool PDFBridgeSumatra::TestFork() {
+	if (this->ForkTested)
+		return true; // Already determined
```

Comment:
```suggestion
		return this->IsForked; // Already determined
```

Why can't this return `this->IsForked`? You don't currently use the return value when calling this method, but to me it makes sense to use this method to check if Sumatra is forked :thinking: 

e.g. `if (this->TestFork()) { /* do stuff */ }`

Should probably document what the return value is for if it's for something else (like indicating an error state)